tutor/src/components/navbar/secondary-toolbar.js

```diff
@@ -12,21 +12,22 @@ const StyledSecondaryToolbar = styled.div`
   position: sticky;
   background: white;
   ${props => !props.renderFn.unpadded && Padding}
-    ${NavbarBottomShadow};
+  ${NavbarBottomShadow};
 `;
 
 @observer
 class SecondaryToolbar extends React.Component {
 
   static propTypes = {
     controls: PropTypes.func.isRequired,
+    className: PropTypes.string,
   }
 
   render() {
-    const { controls } = this.props;
+    const { controls, className } = this.props;
 
     return (
-      <StyledSecondaryToolbar renderFn={controls}>
+      <StyledSecondaryToolbar renderFn={controls} className={cn(className)}>
         {controls()}
```

Comment:
if you're setting classname to just a string there's no reason to use the `cn` `classNames` function